Client/game_sa/CColStoreSA.cpp

```diff
@@ -0,0 +1,152 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        game_sa/CColStoreSA.cpp
+ *
+ *  Multi Theft Auto is available from https://multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+#include "CColStoreSA.h"
+
+void CColStoreSA::Initialise()
+{
+    using Signature = void(__cdecl*)();
+    const auto function = reinterpret_cast<Signature>(0x4113F0);
+    function();
+}
+
+void CColStoreSA::Shutdown()
+{
+    using Signature = void(__cdecl*)();
+    const auto function = reinterpret_cast<Signature>(0x4114D0);
+    function();
+}
+
+void CColStoreSA::BoundingBoxesPostProcess()
+{
+    using Signature = void(__cdecl*)();
+    const auto function = reinterpret_cast<Signature>(0x410EC0);
+    function();
+}
+
+int CColStoreSA::AddColSlot(const char* name)
```

Comment:
Original name is `allocateCollisionFile`.
Just making sure this is the address you want it to be.

Reply:
And where does it say that? Even the Android IDB says AddColSlot.

Reply:
In my IDB.
But no issue, I just wanted to warn you, just in case.